src/views/guidelines/guidelines.scss

```diff
@@ -0,0 +1,40 @@
+@import "../../colors";
+
+#view {
+    .body {
+        #top, #bottom {
```

Comment:
These are declared as classes in the jsx file, but are referenced as ids here. I believe they should be declared as:
```(css)
.body {
    .top,
    .bottom {
.
.
.
}
```